website/src/app/[lang]/[region]/v2/(home)/(sections)/overview.tsx

```diff
@@ -10,11 +10,11 @@ export async function Overview({ lang, region }: DefaultParams) {
 	});
 
 	return (
-		<BaseContainer className="mb-48 flex flex-col items-center justify-center">
-			<Typography size="3xl" weight="medium" color="foreground-dark" className="my-10 w-3/5 text-center">
+		<BaseContainer className="theme-v2 mb-16 flex flex-col items-center justify-center md:mb-48">
+			<Typography size="3xl" weight="medium" className="my-10 w-full text-center md:w-3/5">
 				{translator.t('section-2.title-1')}
 			</Typography>
-			<div className="mb-8 w-3/5 text-center">
+			<div className="mb-8 w-full text-center md:w-3/5">
 				{translator.t<{ text: string; color?: FontColor }[]>('section-2.title-2').map((title, index) => (
 					<Typography
```

Comment:
the `className` prop is probably not needed here

Reply:
Won't we need it for responsiveness?

Reply:
You don't need it because you already set it on the parent element.